packages/near-membrane-base/src/membrane.ts

```diff
@@ -1134,10 +1170,11 @@ export function createMembraneMarshall() {
                 key: string | symbol,
                 receiver: any
             ): ReturnType<typeof Reflect.get> {
-                const { foreignTargetPointer } = this;
-                const safeDesc =
-                    getOwnForeignDescriptor(foreignTargetPointer, shadowTarget, key) ||
-                    getInheritedForeignDescriptor(foreignTargetPointer, key);
+                const safeDesc = lookupForeignDescriptor(
+                    this.foreignTargetPointer,
+                    shadowTarget,
+                    key
+                );
```

Comment:
☝️ Batched these two calls (`getOwnForeignDescriptor` and `getInheritedForeignDescriptor`)  into one